.azure-pipelines/run-test-scheduler-template.yml

```diff
@@ -135,7 +135,7 @@ steps:
       --retry-times ${{ parameters.RETRY_TIMES }} \
       --dump-kvm-if-fail ${{ parameters.DUMP_KVM_IF_FAIL }} \
       --requester "${{ parameters.REQUESTER }}" \
-      --max-execute-seconds ${{ parameters.MAX_EXECUTE_SECONDS }}
+      --max-execute-seconds $((${{ parameters.MAX_RUN_TEST_MINUTES }} * 60))
```

Comment:
If `${{ parameters.MAX_RUN_TEST_MINUTES }}` is a string, then the multiply here looks weird. Although I know that it might work, still I do not feel straight here.